library/NotificationCenter/Queue/QueueManager.php

```diff
@@ -162,12 +161,15 @@ protected function storeFiles(GatewayInterface $gateway, QueuedMessage $queuedMe
 
         // Copy the attachments to the temporary folder
         foreach ($attachments as $index => $originalPath) {
-            $originalPath = str_replace(TL_ROOT . '/', '', $originalPath);
-            $clonePath = $folder->path . '/' . basename($originalPath);
+            $originalPath = '/' === $originalPath[0] ? $originalPath : str_replace(TL_ROOT . '/', '', $originalPath);
```

Comment:
That won't work on Windows. You should use https://packagist.org/packages/webmozart/path-util for that.